script6.js

```diff
@@ -0,0 +1,27 @@
+/* 6.1 */
+
+let Cars = [];
+
+/* 6.2 */
+
+console.log("the value of my variable y will be: []");
+
+/* 6.3 */
+
+console.log(Cars);
+
+/* 6.4 */
+
+let myFavoriteAnimals=['cat', 'dog', 'horse']
+
+/* 6.5 */
+
+console.log(myFavoriteAnimals);
+
+/* 6.6 */
+
+myFavoriteAnimals.push('baby_pig');
+
+/* 6.7 */
+
+console.log(myFavoriteAnimals);
```

Comment:
:+1: good use of camelCase here to name the variable!

Reply:
thanks :)